### PR TITLE
Update Tensorflow to 0.9 release (MLDB-1785)

### PR DESCRIPTION
**Pull request by jeremybarnes**
Pulls in some extra dependencies of TF, and updates one test that asserted on values which have changed slightly.

### CODE REVIEW

File: testing/MLDB-1694-flatten-embeddings.py [hunk 1]
```diff
@@ -43,11 +43,11 @@ def test_prediction_works(self):
...
-                ["softmax.0.866", 0.8080083131790161],
-                ["softmax.0.794", 0.02284531109035015],
-                ["softmax.0.896", 0.009539531543850899],
-                ["softmax.0.849", 0.009413402527570724],
-                ["softmax.0.926", 0.007742570247501135]
+                ["softmax.0.866", 0.8080089092254639],
+                ["softmax.0.794", 0.022845188155770302],
+                ["softmax.0.896", 0.009539488703012466],
+                ["softmax.0.849", 0.009413405321538448],
+                ["softmax.0.926", 0.007742532528936863]
```
Comment (FinchPowers) [inline testing/MLDB-1694-flatten-embeddings.py]: Floating point precision fluctuation?

**Comment by FinchPowers** [comment]
+1

File: testing/MLDB-1694-flatten-embeddings.py (hunk 1 above)
Comment (jeremybarnes) [inline testing/MLDB-1694-flatten-embeddings.py]: Yes, from one version to another of Tensorflow.

**Comment by jeremybarnes** [comment]
@mailletf please give your +1 on this before we merge; in particular we need to make sure that the image classification demo continues to function properly.

**Comment by mailletf** [comment]
everything seems to work. +1

**Comment by jraby** [comment]
Do you want to release first,  then merge this?

**Pull request merged**